tests/renderer_store.test.ts

```diff
@@ -471,45 +472,78 @@ describe("renderer", () => {
     expect(fillStyle).toEqual([{ color: "#DC6CDF", h: 23, w: 96, x: 0, y: 0 }]);
   });
 
-  test("fill style of hovered clickable cells goes over regular fill style", () => {
-    const { drawGridRenderer, model, container } = setRenderer(
-      new Model({ sheets: [{ colNumber: 1, rowNumber: 3 }] })
-    );
+  describe("Table cell hover effect", () => {
+    let model: Model;
+    let ctx: MockGridRenderingContext;
+    let container: DependencyContainer;
+    let drawGridRenderer: (ctx: GridRenderingContext) => void;
+    let fillStyles: any[];
+
     const background = "#DC6CDF";
     const hoverColor = "#017E8414";
```

Comment:
you can use the new constant here 
```suggestion
    const hoverColor = TABLE_HOVER_BACKGROUND_COLOR;
```